Activities/Cryptography/UiPath.Cryptography.Activities/NetCore/ViewModels/KeyedHashFileViewModel.cs

```diff
@@ -49,6 +50,11 @@ public KeyedHashFileViewModel(IDesignServices services) : base(services)
         /// </summary>
         public DesignInArgument<SecureString> KeySecureString { get; set; } = new DesignInArgument<SecureString>();
 
+        /// <summary>
+        /// Switches between 
```

Comment:
?

Reply:
Changed to 'Switches Key as string or secure string '